lsif/src/server.ts

```diff
@@ -217,54 +306,30 @@ async function main(): Promise<void> {
         )
     )
 
-    app.listen(HTTP_PORT, () => {
-        if (LOG_READY) {
-            console.log(`Listening for HTTP requests on port ${HTTP_PORT}`)
-        }
-    })
+    return router
 }
 
 /**
- * Connect and start an active connection to the worker queue. We also run a
- * node-resque scheduler on each server instance, as these are guaranteed to
- * always be up with a responsive system. The schedulers will do their own
- * master election via a redis key and will check for dead workers attached
- * to the queue.
+ * Middleware function used to convert uncaught exceptions into 500 responses.
+ *
+ * @param logger The server's logger instance.
  */
-async function setupQueue(): Promise<Queue> {
-    const [host, port] = REDIS_ENDPOINT.split(':', 2)
-
-    const connectionOptions = {
-        host,
-        port: parseInt(port, 10),
-        namespace: 'lsif',
-    }
-
-    const queue = new Queue({ connection: connectionOptions })
-    queue.on('error', logErrorAndExit)
-    await queue.connect()
-    exitHook(() => queue.end())
-
-    const scheduler = new Scheduler({ connection: connectionOptions })
-    scheduler.on('error', logErrorAndExit)
-    await scheduler.connect()
-    exitHook(() => scheduler.end())
-    await scheduler.start()
+function errorHandler(
+    logger: Logger
+): (e: any, req: express.Request, res: express.Response, next: express.NextFunction) => void {
+    return (e: any, req: express.Request, res: express.Response, next: express.NextFunction): void => {
+        if (res.headersSent) {
+            return next(e)
+        }
 
-    return queue
-}
+        if (e && e.status) {
+            res.status(e.status).send({ message: e.message })
+            return
+        }
 
-/**
- * Middleware function used to convert uncaught exceptions into 500 responses.
- */
-function errorHandler(err: any, req: express.Request, res: express.Response, next: express.NextFunction): void {
-    if (err && err.status) {
-        res.status(err.status).send({ message: err.message })
-        return
+        logger.error('uncaught exception', { error: e })
```

Comment:
```suggestion
        logger.error('Uncaught exception', { error: e })
```

Convention (opposite of Go) is to upper case logs (e.g. native `Unhandled exception` logs)

Reply:
Should this change be made to all logs?

Reply:
Ideally yes. It's not as important though, mainly wanted to raise this for future code.